data/scripts/debug.inc

```diff
@@ -14,6 +14,7 @@ Debug_CheatStart::
 	setflag FLAG_RESCUED_BIRCH
 	setflag FLAG_HIDE_ROUTE_101_BIRCH_ZIGZAGOON_BATTLE
 	setflag FLAG_ADVENTURE_STARTED
+	setflag FLAG_BADGE08_GET
```

Comment:
Intentional change?

Reply:
Yeah, it's needed for testing because of obedience but I didn't want to make a lone PR for it.

Reply:
Removed it from this PR since a different PR was merged that has it lol